packages/nextjs/src/config/templates/serverComponentWrapperTemplate.ts

```diff
@@ -2,8 +2,10 @@ import * as Sentry from '@sentry/nextjs';
 import type { WebFetchHeaders } from '@sentry/types';
 // @ts-expect-error Because we cannot be sure if the RequestAsyncStorage module exists (it is not part of the Next.js public
 // API) we use a shim if it doesn't exist. The logic for this is in the wrapping loader.
+// biome-ignore lint/nursery/noUnusedImports: Biome doesn't understand the shim with variable import path
```

Comment:
Can you also open an issue to Biome for this?

Reply:
Done https://github.com/biomejs/biome/issues/1269

Reply:
Turns out that it's a typescript bug, so i refactored this code a little.